src/main/java/uk/gov/justice/digital/delius/controller/secure/OffendersResource.java

```diff
@@ -330,5 +330,28 @@ public OffenderDetail getOffenderDetailByNomsNumber(final @PathVariable("nomsNum
                 .map(convictionService::convictionsFor)
                 .orElseThrow(() -> new NotFoundException(String.format("Offender with crn %s not found", crn)));
     }
+
+    @ApiOperation(value = "Return the conviction (AKA Delius Event) for a conviction ID and a CRN")
+    @ApiResponses(
+        value = {
+            @ApiResponse(code = 200, message = "OK", response = Conviction.class, responseContainer = "List"),
```

Comment:
I guess `responseContainer` attribute isn't needed? Not sure what impact it has keeping it.

Reply:
Copy and paste from previous method. Yes.